src/index.html

```diff
@@ -8,9 +8,36 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./main.scss">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-form search-form--long"
+      data-qa="big"
+      >
+      <label class="search-form__label search-form__label--long">
```

Comment:
```suggestion
      class="search-form"
      data-qa="big"
      >
      <label class="search-form__label">
```
check this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
and fix all similar cases
![Screenshot from 2023-09-09 17-57-50](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/8d3b1b30-c173-4181-a506-503bcca3bea1)